documentation/doxygen/filter.cxx

```diff
@@ -68,8 +68,16 @@
 ///    To generate the corresponding jupyter notebook. In case the tutorial does
 ///    not generate any graphics output, the option `-nodraw` should be added.
 ///
+///  5. `\preview`
+///    To display the first image generated by the tutorial as an icon in the list of tutorials.
+///
```

Comment:
Are these changes part of this PR? It looks unrelated

Reply:
It is a missing comment . I saw it when updating the doc.